Having,setup,the,settings.py,file,to,recognize,an,installed,app,of,',blog,',",",I,still,get,this,error,No,module,named,blog,But,the,models,saves,in,',blog,',folder,are,installed,on,the,sqlite3,db,using,syncdb,.,Any,idea,why,this,happens,and,how,to,fix,this,error,?
